document_page/__manifest__.py

```diff
@@ -43,7 +32,5 @@
     'demo': [
         'demo/document_page.xml'
     ],
-    'installable': False,
```

Comment:
Personally, keeping the install key makes it much easier to use grep to list installable/non installable modules. 
But that is just a personal preference and I won't pick on this then.